Day_2_Software_engineering_best_practices/README.md

```diff
@@ -124,12 +124,18 @@ Some theory:
 
 **Exercise:**
 
-* make package with setup.py
+We created `setup.py` and `spectral_analysis/version.py` to make the a
```

Comment:
to avoid having `version` (which you should not use, you already have `__version__` in your namespace